assets/js/modules/analytics/components/common/AdsConversionIDTextField.js

```diff
@@ -91,7 +91,7 @@ export default function AdsConversionIDTextField() {
 			</TextField>
 
 			<p>
-				{ __( 'Insert your Ads Conversion ID here if you want Site Kit to place the snippet on your site', 'google-site-kit' ) }
+				{ __( 'If you’re using Google Ads, insert your Ads conversion ID if you’d like Site Kit to place the snippet on your site', 'google-site-kit' ) }
```

Comment:
@felixarntz looks like maybe a good place to add a "Learn more" link – what do you think?

Reply:
Yes, I was thinking that too, but we don't really have an appropriate one at this point. Maybe this can be improved with #3282 if we know of something by then.